Applications/DataExplorer/DataView/MeshView.cpp

```diff
@@ -244,9 +244,11 @@ void MeshView::openAddLayerDialog()
         return;
     }
 
-    double const thickness (dlg.getThickness());
-    std::unique_ptr<MeshLib::Mesh> result(MeshLib::addLayerToMesh(
-        *mesh, thickness, dlg.getName(), dlg.isTopLayer()));
+    bool const take_over_material_ids = false;
```

Comment:
Take over the world, that's right. Here pass material ids, or copy material ids would be better, before we take over the world. It's a funny evening today ;)

Reply:
Renamed as suggested.